Dungeon Crawler/Assets/Scripts/BattleManager.cs

```diff
@@ -20,9 +22,6 @@ void Awake()
     {
         playerInputActions = new PlayerInputActions();
```

Comment:
We can remove everything regarding PlayerInputActions completely from the BattleManager, as it will not be used here anymore.

Reply:
Pushed a fix which removes all unneccesary lines.